Detections/CommonSecurityLog/Wazuh-Large Number of Web errors from an IP

```diff
@@ -0,0 +1,28 @@
+id:
+name: Wazuh - Large Number of Web errors from an IP
+description: |
+  'Identifies instances where Wazuh logged over 400 '403' Web Errors from one IP Address.'
```

Comment:
I think it would be a good idea to link to the documentation on how to onboard this data into Azure Sentinel as it doesn't have a specific default connector - https://github.com/wazuh/wazuh-documentation/blob/master/source/azure/monitoring%20activity.rst